packages/url/src/is-file-url.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Determines whether the given string looks like a file URL.
+ *
+ * @param {string} url The string to scrutinise.
+ *
+ * @example
+ * ```js
+ * const isFileURL = isFileURL( 'file:///file.txt' ); // true
+ * ```
+ *
+ * @return {boolean} Whether or not it looks like a file URL.
+ */
+export function isFileURL( url ) {
+	return url !== undefined && url.startsWith( 'file:' );
```

Comment:
There's a `getProtocol` function in this module already. Shouldn't we be using it here? Shouldn't it be used _instead_ of adding a new function?

i.e. What's the difference between `isFileURL( url )` and `getProtocol( url ) === 'file:'` ?

Reply:
I prefer to have an explicit method for this, but I can use the getProtocol method inside this function. What do you say @aduth ?

Reply:
Is the preference on the basis that this is a common need? I just wonder how we answer question of which protocols are "deserving" of their own functions. Should we have `isTelephoneURL`? `isSSHURL`? `isMailToURL`? One of the advantages of `getProtocol` is that we don't have to answer that question, nor do we leave any uncertainty to the consuming developers which and why some protocols have specific functions. Is it really enough of a pain point in using `getProtocol`?

Reply:
Ok me @aduth talked and decided to refactor the code to use getProtocol to simplify the API and have a more generic approach.